src/js/components/Image/__tests__/Image-test.js

```diff
@@ -96,5 +96,5 @@ test('Image onError', () => {
     fireEvent(getByAltText('test'), new Event('error'));
   });
 
-  expect(onError).toHaveBeenCalledTimes(1);
+  expect(onError).toHaveBeenCalledTimes(0);
```

Comment:
```suggestion
  expect(onError).toHaveBeenCalledTimes(1);
```
We should still expect onError to be called 1 time.